src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -98,10 +98,11 @@ public function load(string $path, string ...$extraPaths): void
      * .env.local is always ignored in test env because tests should produce the same results for everyone.
      * .env.dist is loaded when it exists and .env is not found.
      *
-     * @param string      $path       A file to load
-     * @param string|null $envKey     The name of the env vars that defines the app env
-     * @param string      $defaultEnv The app env to use when none is defined
-     * @param array       $testEnvs   A list of app envs for which .env.local should be ignored
+     * @param string      $path                 A file to load
+     * @param string|null $envKey               The name of the env vars that defines the app env
+     * @param string      $defaultEnv           The app env to use when none is defined
+     * @param array       $testEnvs             A list of app envs for which .env.local should be ignored
+     * @param bool        $overrideExistingVars Whether existing environment variables set by the system must be overridden. Variables defined in .env files won't be overridden.
```

Comment:
I'm not sure about the last part of the sentence (+ we don't use final dots for these entries)
```suggestion
     * @param bool        $overrideExistingVars Whether existing environment variables set by the system must be overridden
```

Reply:
I updated and added a link to the doc, now that's documented (like it's been done [here](https://github.com/symfony/symfony/pull/51974/files#diff-deee1a5892810656ac3eb9f0835a435cd95b01bc7bb79fccefde595c6428f992R37) for example)